### ISSUE TITLE
Ctrl-^ does not work after using vimtex-toc

### ISSUE THREAD

**Issue by kiryph** [lervag/vimtex#549]
### Explain the issue

I use very often the key combination `Ctrl-^` to switch back to the last file (in vim called the alternate file).

However, when I use vimtex-toc to jump to a different chapter which is located in a different file, I cannot use `Ctrl-^` to jump back. Following error message is shown:

```
E23: No alternate file
```

In particular, I use the setting `let g:vimtex_index_split_pos = 'full'`.

If you can't reproduce this easily, I am happy to provide a full MWE.

---
I know that I have submitted quite a few issues in the last time. I hope you don't mind. Presumably, you can close #546 without investigation since IMHO a solution depends on other projects before making this working with vimtex.

**Comment by kiryph** [comment]
I fixed it for me the following way:

```diff
@@ -167,7 +167,7 @@ function! s:actions_activate(close) dict "{{{1
   endif

   " Open file buffer
-  execute cmd bnr
+  execute 'keepalt' cmd bnr

   " Go to entry line
   call setpos('.', [0, entry.line, 0, 0])
```

**Comment by lervag** [comment]
Thanks, I fixed this as you suggested, but also fixed a problem where `bufwipeout` errored because the buffer no longer existed. You probably didn't see this because you had `set hidden`, but I found it while using a minimal vimrc.

I am just happy that you are opening issues for bugs and suggestions for improvements. Although I have quite a lot of things going on in my life (mostly family life with two kids), so I don't always have the time to fix things quickly. So if you don't mind that it may take some time for me to handle your issues, then we're both happy. :)
